### ISSUE TITLE
Run tests on arbitrum

### ISSUE THREAD

**Issue by philippecamacho** [EspressoSystems/cape#1229]

**Comment by philippecamacho** [comment]
Documentation for running tests on arbitrum done in https://github.com/EspressoSystems/cape/commit/0bc3adee84c3274e7a09a0a945f17262c6e92255

**Comment by philippecamacho** [comment]
So far some tests are passing, many failing:
```

test result: FAILED. 20 passed; 74 failed; 0 ignored; 0 measured; 0 filtered out; finished in 114.19s

error: test failed, to rerun pass '-p cap-rust-sandbox --lib'

```

The main reason is a "nonce too low error" that happens in many places, e.g.:

`thread 'cape_e2e_test_transfer::test_2user_and_submit' panicked at 'called `Result::unwrap()` on an `Err` value: (code: -32000, message: nonce too low: address 0x2FB18F4b4519a5fc792cb6508C6505675BA659E9, tx: 29 state: 30, data: None)', contracts/rust/src/deploy.rs:44:6
`

**Comment by sveitser** [comment]
@philippecamacho Did you specify `--test-threads 1`? This is required if we don't have our local dev chain with an unlocked account and therefore can't send an unsigned transaction to deploy the contract.

**Comment by sveitser** [comment]
Closed by #1230
